infrastructure/zk/src/config.ts

```diff
@@ -17,7 +17,7 @@ const CONFIG_FILES = [
     'gateway_watcher.toml',
     'fee_ticker.toml',
     'misc.toml',
-    'dev_liquidity_token_watcher.toml',
+    'dev_ticker.toml',
```

Comment:
How the dev_ticker differs from fee_ticker? Shouldn't we use one single file to configure the ticker?

Reply:
fee_ticker contains configurations for production, whereas dev_ticker for ... well ... dev